docs/enable_disable_module.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Enable and disable a Kyma module
+---
+
+## Overview
+
+Your cluster comes with the Kyma custom resource (CR) already installed. It collects all metadata about the cluster, such as enabled modules, their statuses, or synchronization, using Lifecycle Manager. Lifecycle Manager uses `moduletemplate.yaml` to enable or disable modules on your cluster. 
+
+## Procedure
+
+> **NOTE:** To quickly enable a module, go to your **Cluster Details** view and select **Add Module**. Then, choose the channel of your choice and click **Add** to choose and upload your module.
+
+> **TIP:** You can also configure your Kyma CR to enable a module manually. For more details, see [Kyma CR](https://github.com/kyma-project/lifecycle-manager/blob/main/docs/technical-reference/api/kyma-cr.md).
+
+<div tabs name="steps" group="enable-module">
```

Comment:
if this is called "enable module", why does it also contain instructions to disable it? Will readers find it? Why not two procedures (each with tabs for CLI and GUI). The "disable" procedure will be very short, but still it fulfils a separate user goal.

Reply:
Sure, good idea. I'll change that.